src/Dom/Browser.Dom.fs

```diff
@@ -1385,8 +1385,32 @@ type [<AllowNullLiteral>] HTMLElement =
     abstract insertAdjacentHTML: where: string * html: string -> unit
     abstract insertAdjacentText: where: string * text: string -> unit
     abstract scrollIntoView: ?top: bool -> unit
+    abstract scrollIntoView: ?scrollIntoViewOptions: ScrollIntoViewOptions -> unit
```

Comment:
I just noticed that, because we already have a `scrollIntoView` overload with an optional parameter, if we add a new one, when we don't have parameters (e.g. `x. scrollIntoView()`) F# won't know how to resolve the overload and will throw an error. It seems this can be fixed by making `scrollIntoViewOptions` **not** optional in the second overload ([example](https://fable.io/repl/#?code=C4TwDgpgBAygxgJwPYBsUEkB2wkDUCWEA7gPJjD5KYDOUAvAFBTNT7YQIBmAhnNBJgAmDBqEhQAYkiT0mLbgCNqwBL2BRqiVBmx5CRAFxQA-DjBGF0lFAC0APigBXTPmBzmi5arjrNyNFg4BMRGftqBesRkFFTURvD+OkH60ZQ0tg7OriIoEOrAEMoAFJxGUkgAlLIsUJwAdGEBusFERRUMQA&html=DwCwLgtgNgfAUAAiQ0BTAhgE3s3KKpjoLhgAOAtKgI4CuAlgG4C8ARAMID2AdmKrxQAqATzKpWCAMY8+vNnwAeYAPThoAbkkh0AJwDOhZrTAAzCgA5WOZMFUZsiGwCNOmYdaS2Xb+LbWwgA&css=Q)).

```fsharp
abstract scrollIntoView: ?scrollIntoViewOptions: ScrollIntoViewOptions -> uni
```

Reply:
Good catch, I didn't test the whole code and only made sure the compiler was builder the library.